packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/add-notebook.html

```diff
@@ -0,0 +1,262 @@
+<section>
+    <!-- page header -->
+    <oui-header
+        class="mb-2"
+        data-heading="{{ ::'data_processing_add_notebook_title' | translate }}"
```

Comment:
we have mainly this syntax on the rest of page:
```suggestion
        data-heading="{{:: 'data_processing_add_notebook_title' | translate }}"
```

Reply:
Fixed :)